ert/data/record/_record.py

```diff
@@ -59,6 +59,13 @@ class RecordType(str, Enum):
 
 
 class Record(ABC):
+    """The :class:`Record` class is an abstract class that
+    includes record data and the record type.
```

Comment:
Can we say something about what we intend to use this kind of object for? 

ala
"A record is a data container designed to represent units of input and output data to stages".  (feel free to elaborate and make more precise)